doc/browsing_cubes/intro.rst

```diff
@@ -1,9 +1,54 @@
 Browsing Cubes
 ==============
 
-This is a description of how you can use the cube_browser package to look at cubes and make plots and sliders without having to make any widgets.
+The Cube Browser package offers an API which you can use to visualize your cubes in a Jupyter notebook.
```

Comment:
What's an API? This might be a good use for an [abbreviation](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-abbr) or a short description of what an API is.